astropy/utils/iers/iers.py

```diff
@@ -351,8 +371,20 @@ def _check_interpolate_indices(self, indices_orig, indices_clipped, max_input_mj
         class because it has different requirements.
         """
         if np.any(indices_orig != indices_clipped):
-            raise IERSRangeError('(some) times are outside of range covered '
-                                 'by IERS table.')
+            if conf.iers_degraded_accuracy == 'error':
+                msg = ('(some) times are outside of range covered by IERS table. Cannot convert '
+                       'with full accuracy. To allow conversion with degraded accuracy '
+                       'set `astropy.utils.iers.conf.iers_degraded_accuracy` '
+                       " to 'warn' or 'silent'")
+                raise IERSRangeError(msg)
+            elif conf.iers_degraded_accuracy == 'warn':
+                # No IERS data covering the time(s) and user requested a warning.
+                msg = ('(some) times are outside of range covered by IERS table, '
+                       'accuracy is degraded.')
+                warn(msg, IERSDegradedAccuracyWarning)
+            else:
+                # No IERS data covering the time(s) and user is OK with no warning.
+                pass
```

Comment:
The `pass` seems redundant.
```suggestion
            # No IERS data covering the time(s) and user is OK with no warning.
```